doc/procs/validate.md

```diff
@@ -0,0 +1,33 @@
+qc::validate
+===========
+
+part of [Docs](../index.md)
+
+Usage
+-----
+`validate dict`
+
+Description
+-----------
+Validates a given dictionary against the data model and sets up the record in the global data structure.
```

Comment:
Validates a given dictionary of name value pairs against the data model and sets up the record in the global json-payload.